src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java

```diff
@@ -34,10 +34,9 @@ public RedirectView home() {
      */
     @Operation(summary = "Return OK if application is started", security = {@SecurityRequirement(name = "ApiKey"), @SecurityRequirement(name = "Authorization")}, tags = {"Home"})
     @ApiResponses(value = {
-            @ApiResponse(responseCode = "200", description = "OK."),
-            @ApiResponse(responseCode = "403", description = "Forbidden client error status.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema())),
-            @ApiResponse(responseCode = "429", description = "Too many requests", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema())),
-            @ApiResponse(responseCode = "500", description = "Service unavailable.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class)))})
+            @ApiResponse(responseCode = "200", description = "OK"),
+            @ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "500",description = "Service unavailable", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class)))})
```

Comment:
if you remove `429` as `400` too, because manage it on apim policy, we need to add to our infra something [like that LimitCallRate](https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#LimitCallRate)

are'u agree ?

Reply:
Done: https://github.com/pagopa/pagopa-infra/pull/93